README.md

```diff
@@ -3,7 +3,44 @@
 Tool for semantic **Diff**erence of **Ke**rnel functions, **m**odules, and
 **p**arameters.
 
+## Installation
+
+There are two options to install the project, either build from source or use a
+prepared RPM package for Fedora.
+
+### Install from source
+
+Currently, DiffKemp runs on Linux and needs the following software installed:
+* LLVM (supported versions are 5, 6, 7 and 8)
```

Comment:
Maybe it would be helpful for the user to mention that DiffKemp also required Clang, since Clang is usually not included in LLVM packages, although it is build together with LLVM.